src/app/screens/LNURLPay.tsx

```diff
@@ -193,7 +193,7 @@ function LNURLPay(props: Props) {
           {[
             ["Send payment to", details.domain],
             ...formattedMetadata(),
-            ["Amount (Satoshi)", renderAmount()],
+            ["Amount (sat)", renderAmount()],
```

Comment:
```suggestion
            ["Amount (Satoshi)", renderAmount()],
```

let's keep amount here as of @MoritzKa suggestion